tools/arriba.wdl

```diff
@@ -0,0 +1,315 @@
+## [Homepage](https://arriba.readthedocs.io/en/latest/)
+#
+# SPDX-License-Identifier: MIT
+# Copyright St. Jude Children's Research Hospital
+version 1.1
+
+task arriba {
+    meta {
+        description: "Run Arriba structural variant caller on a STAR-produced BAM file."
```

Comment:
Question: Does it _have_ to be STAR-produced? How exhaustive are Arriba's assumption that it's STAR-produced? I would assume to get data from another aligner through Arriba you'd have to do some post-processing. Is it enough post-processing that we can say "no one in their right mind would do it that way"?

Note: I didn't check if Arriba docs specify the STAR assumption. If they do, the above is fine as it is and you don't have to dig for more information. But I would still be curious about the answer to those questions haha.

Reply:
Interesting, I'm glad I asked! How I would document this, (though I don't think this is the only right way), is to leave the `description` vague and say it requires an RNA-Seq BAM. Then I'd use the `help` key to explain a summary of what you found. ("Typical is STAR, but DRAGEN and anything SPEC compliant for features XYZ will work")